src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -37,42 +37,12 @@ class ORMQueryBuilderLoader implements EntityLoaderInterface
     /**
      * Construct an ORM Query Builder Loader.
      *
-     * @param QueryBuilder|\Closure $queryBuilder The query builder or a closure
-     *                                            for creating the query builder.
-     *                                            Passing a closure is
-     *                                            deprecated and will not be
-     *                                            supported anymore as of
-     *                                            Symfony 3.0.
-     * @param ObjectManager         $manager      Deprecated.
-     * @param string                $class        Deprecated.
+     * @param QueryBuilder $queryBuilder The query builder for creating the query builder.
      *
      * @throws UnexpectedTypeException
      */
-    public function __construct($queryBuilder, $manager = null, $class = null)
+    public function __construct(QueryBuilder $queryBuilder)
```

Comment:
this means that people will get a PHP fatal error instead of an `UnexpectedTypeException`, but I guess that is not a problem?